index.html

```diff
@@ -85,10 +85,10 @@ <h3 class="section-box__title--left">Chosen products:</h3>
                 <li
                   class="chosen-product-list__item"
                 >
-                  <div class="chosen-product-list__item--title">
+                  <div class="chosen-product-list__item chosen-product-list__item--title">
                     Apple:
                   </div>
-                  <div class="chosen-product-list__item-description__wrapper">
+                  <div class="chosen-product-list__item-description--wrapper">
```

Comment:
Ta klasa jest dalej źle, nie może to być modyfikator, bo:
- nie ma klasy bazowej, którą ma modyfikować
- sposób użycia - wrapowanie innych elementów - wskazuje, że to jest po prostu osobny element.

Więć poprawnie w tym wypadku będzie nazwać te klasę `chosen-product-list__item-description-wrapper`, `__item-description-wrapepr` to nazwa tego elementu.